### PR TITLE
fix(theme): improve skeleton for the category sidebar

### PR DESCRIPTION
**Pull request by bartoszherba**
## Description
- improve skeleton for the category sidebar

## Related Issue
-

## How Has This Been Tested?
-

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/modules/catalog/category/components/sidebar/CategorySidebar.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <SkeletonLoader
     :is-loading="isLoading"
-    :height="'50vw'"
+    :height="'500px'"
```
Comment (sethidden) [inline packages/theme/modules/catalog/category/components/sidebar/CategorySidebar.vue:4]:
```suggestion
    height="500px"
```
Super nitpick, but now that you've touched it :P

File: packages/theme/modules/catalog/pages/default.vue [hunk 2]
```diff
@@ -14,8 +14,13 @@
       :pagination="pagination"
     />
     <div class="main section">
+      <SkeletonLoader
+        v-if="$fetchState.pending"
+        class="sidebar desktop-only"
+        :height="'500px'"
```
Comment (sethidden) [inline packages/theme/modules/catalog/pages/default.vue:20]:
```suggestion
        height="500px"
```

**Review by sethidden — APPROVED** [review]

File: packages/theme/modules/catalog/category/components/sidebar/CategorySidebar.vue (hunk 1 above)
Comment (sethidden) [reply packages/theme/modules/catalog/category/components/sidebar/CategorySidebar.vue:4]: Feel free to commit with 
image

Comment (bartoszherba) [reply packages/theme/modules/catalog/category/components/sidebar/CategorySidebar.vue:4]: Not really, I am not sure how it suppose to work?

**Review by Frodigo — APPROVED** [review]

**Pull request merged**